src/common/notifications.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include <string_view>
+#include <vector>
+
+namespace notif
+{
+    void register_background_toast_handler();
+
+    void show_toast(std::wstring_view message);
+    void show_toast_background_activated(std::wstring_view background_handler_id, std::wstring_view message, std::vector<std::wstring_view> button_labels);
```

Comment:
What's this `background_handler_id` and why it is not in `show_toast`?

Reply:
@bzoz added a documentation to `/doc/devdocs/common.md`